app/src/main/java/org/mozilla/fenix/library/history/HistoryAdapter.kt

```diff
@@ -83,40 +79,6 @@ class HistoryAdapter(
     }
 
     companion object {
-        private const val zeroDays = 0
```

Comment:
Why move it away from here? It looks pretty related to `HistoryItemTimeGroup`

Reply:
As agreed, I'll move it to `HistoryItemTimeGroup`